### PR TITLE
[mainnet] Deploy Smart Escrow Contract

### PR DESCRIPTION
**Pull request by anikaraghu**
Deploy the [SmartEscrow contract](https://github.com/base-org/contracts/blob/main/src/smart-escrow/SmartEscrow.sol) which now has alignment across Base and OP and gone through audit

### CODE REVIEW

File: mainnet/2024-02-21-setup-smart-escrow/.env [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+DEPLOYER=0xbebe472f467888b197b90693b8852ad12a50b261
+BENEFACTOR= # TODO
+BENEFICIARY=0x07114fF6F815113729b579B4A233192BAEF0e0E18
+BENEFACTOR_OWNER=0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0
+BENEFICIARY_OWNER=0x6e1DFd5C1E22A4677663A81D24C6BA03561ef0f6
+NESTED_SAFE=0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940
+START=1720674000
+END=1870572600
+VESTING_PERIOD_SECONDS=7889400
```
Comment (anikaraghu) [inline mainnet/2024-02-21-setup-smart-escrow/.env:16]: https://docs.google.com/spreadsheets/d/1nIh13ccelNtfmiIptucvDPtA8e94HPjQtxR9cARoYZM/edit#gid=0

File: mainnet/2024-02-21-setup-smart-escrow/.env [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+BASE_CONTRACTS_COMMIT=a147139671c09923f78ae46a6ebedc91209bb076
+
+OP_TOKEN=0x4200000000000000000000000000000000000042
+
+DEPLOYER=0xbebe472f467888b197b90693b8852ad12a50b261
+BENEFACTOR= # TODO
+BENEFICIARY=0x07114fF6F815113729b579B4A233192BAEF0e0E18
+BENEFACTOR_OWNER=0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0
+BENEFICIARY_OWNER=0x6e1DFd5C1E22A4677663A81D24C6BA03561ef0f6
+NESTED_SAFE=0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940
```
Comment (anikaraghu) [inline mainnet/2024-02-21-setup-smart-escrow/.env:13]: https://optimistic.etherscan.io/address/0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940

File: mainnet/2024-02-21-setup-smart-escrow/.env [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+OP_COMMIT=3580bf1b41d80fcb2b895d5610836bfad27fc989
+BASE_CONTRACTS_COMMIT=a147139671c09923f78ae46a6ebedc91209bb076
+
+OP_TOKEN=0x4200000000000000000000000000000000000042
+
+DEPLOYER=0xbebe472f467888b197b90693b8852ad12a50b261
+BENEFACTOR= # TODO
+BENEFICIARY=0x07114fF6F815113729b579B4A233192BAEF0e0E18
+BENEFACTOR_OWNER=0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0
```
Comment (anikaraghu) [inline mainnet/2024-02-21-setup-smart-escrow/.env:11]: OP foundation multisig: https://optimistic.etherscan.io/address/0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0#readProxyContract

File: mainnet/2024-02-21-setup-smart-escrow/.env [hunk 4]
```diff
@@ -0,0 +1,18 @@
+OPTIMISM_RPC_URL=https://mainnet.optimism.io
+
+OP_COMMIT=3580bf1b41d80fcb2b895d5610836bfad27fc989
+BASE_CONTRACTS_COMMIT=a147139671c09923f78ae46a6ebedc91209bb076
+
+OP_TOKEN=0x4200000000000000000000000000000000000042
+
+DEPLOYER=0xbebe472f467888b197b90693b8852ad12a50b261
```
Comment (anikaraghu) [inline mainnet/2024-02-21-setup-smart-escrow/.env:8]: https://confluence.coinbase-corp.com/display/DEVP/%5BBase+Key+Addresses%5D+Mainnet

File: mainnet/2024-02-21-setup-smart-escrow/.env [hunk 5]
```diff
@@ -0,0 +1,18 @@
+OPTIMISM_RPC_URL=https://mainnet.optimism.io
+
+OP_COMMIT=3580bf1b41d80fcb2b895d5610836bfad27fc989
+BASE_CONTRACTS_COMMIT=a147139671c09923f78ae46a6ebedc91209bb076
+
+OP_TOKEN=0x4200000000000000000000000000000000000042
```
Comment (anikaraghu) [inline mainnet/2024-02-21-setup-smart-escrow/.env:6]: https://optimistic.etherscan.io/address/0x4200000000000000000000000000000000000042

File: mainnet/2024-02-21-setup-smart-escrow/.env [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+DEPLOYER=0xbebe472f467888b197b90693b8852ad12a50b261
+BENEFACTOR=0x2501c477d0a35545a387aa4a3eee4292a9a8b3f0
```
Comment (anikaraghu) [inline mainnet/2024-02-21-setup-smart-escrow/.env:9]: should send tokens back to where they came from - see txs of tokens comingn in on https://optimistic.etherscan.io/address/0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940#tokentxns

File: mainnet/2024-02-21-setup-smart-escrow/.env [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+BENEFICIARY= # TODO
+BENEFACTOR_OWNER=0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0
+BENEFICIARY_OWNER=0x6e1DFd5C1E22A4677663A81D24C6BA03561ef0f6
+NESTED_SAFE=0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940
+START=1720674000
+END=1870572600
+VESTING_PERIOD_SECONDS=7889400
+INITIAL_TOKENS=17895697000000000000000000
+VESTING_EVENT_TOKENS=4473924000000000000000000
```
Comment (anikaraghu) [inline mainnet/2024-02-21-setup-smart-escrow/.env:18]: https://docs.google.com/document/d/1a6V3YyDQEMaLiaaqIvrcCRfnqcAjyxkhVSQCrv3BxKs/edit#heading=h.66mvt9n8oxmj

File: mainnet/2024-02-21-setup-smart-escrow/.env [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+OP_TOKEN=0x4200000000000000000000000000000000000042
+
+DEPLOYER=0xbebe472f467888b197b90693b8852ad12a50b261
+BENEFACTOR=0x2501c477d0a35545a387aa4a3eee4292a9a8b3f0
+BENEFICIARY=0x635Fb974F09B269Bc750bF96338c29cF41430125
+BENEFACTOR_OWNER=0x2501c477D0A35545a387Aa4A3EEe4292A9a8B3F0
+BENEFICIARY_OWNER=0x6e1DFd5C1E22A4677663A81D24C6BA03561ef0f6
+NESTED_SAFE=0x0a7361e734cf3f0394B0FC4a45C74E7a4Ec70940
+START=1720674000
+END=1878462000
```
Comment (anikaraghu) [inline mainnet/2024-02-21-setup-smart-escrow/.env:15]: https://docs.google.com/spreadsheets/d/1nIh13ccelNtfmiIptucvDPtA8e94HPjQtxR9cARoYZM/edit#gid=0

File: mainnet/2024-02-21-setup-smart-escrow/.env [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+VESTING_PERIOD_SECONDS=7889400
```
Comment (anikaraghu) [inline mainnet/2024-02-21-setup-smart-escrow/.env:16]: https://docs.google.com/spreadsheets/d/1nIh13ccelNtfmiIptucvDPtA8e94HPjQtxR9cARoYZM/edit#gid=0

**Review by stevieraykatz — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for stevieraykatz @ 2024-02-27 23:38:05 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Review by stevieraykatz — APPROVED** [review]

**Review by stevieraykatz — APPROVED** [review]

**Pull request merged**